atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/metrics/SweepOutcomeMetricsTest.java

```diff
@@ -45,22 +45,24 @@ public void setup() {
         legacyMetrics = SweepOutcomeMetrics.registerLegacy(metricsManager);
         targetedSweepMetrics = TargetedSweepMetrics
                 .create(metricsManager, mock(TimelockService.class), new InMemoryKeyValueService(true), Long.MAX_VALUE);
+        SweepOutcomeMetrics.LEGACY_OUTCOMES.forEach(outcome -> {
+            assertThat(metricsManager).hasNotRegisteredLegacyOutcome(outcome);
+        });
+        SweepOutcomeMetrics.TARGETED_OUTCOMES.forEach(outcome -> {
+            assertThat(metricsManager).hasNotRegisteredTargetedOutcome(CONSERVATIVE, outcome);
+            assertThat(metricsManager).hasNotRegisteredTargetedOutcome(THOROUGH, outcome);
+        });
```

Comment:
The assertions are now run before every test method - this is not too problematic, but I'd prefer to have a separate test that doesn't do anything and just checks these.

Reply:
Done